tests/testthat/test-init.R

```diff
@@ -56,17 +56,48 @@ testthat::test_that("init throws when an empty `data` is used", {
   )
 })
 
-testthat::test_that("init throws warning when datanames in modules incompatible w/ datanames in data", {
-  testthat::local_mocked_bindings(log_warn = warning, .package = "logger")
+testthat::test_that(
+  "init throws warning when datanames in modules incompatible w/ datanames in data and there is no transformers",
+  {
+    testthat::local_mocked_bindings(log_warn = warning, .package = "logger")
```

Comment:
Is it needed? I thought `tests/testthat/setup-logger.R` turns off logger in tests

Reply:
Yeah, looks like some old remaining. This works without mocking. I though it is needed to see the actual warning message. Just removed those https://github.com/insightsengineering/teal/pull/1319